docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -1814,3 +1814,35 @@ The connector supports redirection from Iceberg tables to Hive tables with the
 
 The connector supports configuring and using [file system
 caching](/object-storage/file-system-cache).
+
+### Iceberg metadata caching
+
+When file system caching from `fs.cache.enabled` is enabled, Iceberg metadata is
+cached on local disks using the files system caching implementation. The Iceberg
```

Comment:
I think this is wrong now ... here is what I understood from @raunaqmorarka 

fs.cache.enabled=true - metadata (and data) is cached with the file system caching (so on disk or whatever is configured there), AND .. memory metadata caching is deactivated

fs.cache.enabled=false - memory metadata caching is automatically activated, and uses coordinator memory only (hence planning and such is done on coordinator so thats where that metadata is needed)

Reply:
@mosabua yes the above is correct, but I think what is written now also conveys the same thing. Maybe the sentence in the warning should move up to make that clearer.

Reply:
I think the current wording is wrong order of things and should be reworded.